pkg/apis/minio.min.io/v1/helper.go

```diff
@@ -83,6 +87,49 @@ func GetNSFromFile() string {
 	return string(namespace)
 }
 
+// GetTagFromContainer returns the tag value as received from `minio -v` command
+func GetTagFromContainer(image string) (string, error) {
+	ctx := context.Background()
+	cli, err := client.NewEnvClient()
```

Comment:
For a given tenant there might be a private registry config in the CRD that we would need to honor